packer/gotool_test.go

```diff
@@ -0,0 +1,31 @@
+package packer
+
+import "testing"
+
+func TestPkgBasename(t *testing.T) {
+	f := func(p Pkg, wantBasename string) {
```

Comment:
Can you turn this into a table-driven test please? See https://go.dev/wiki/TableDrivenTests

Reply:
Done (I find [f-tests](https://medium.com/@valyala/f-tests-as-a-replacement-for-table-driven-tests-in-go-8814a8b19e9e) sometimes easier to read, but since table-driven tests are already used in the code, it is better to keep it consistent, as you proposed :)